sdk/storage/azure-storage-blobs/CHANGELOG.md

```diff
@@ -2,10 +2,14 @@
 
 ## 1.0.0-beta.3 (2020-10-13)
 
+### New Features
+
 * Bump up API version to 2020-02-10
 * Support for Last Accessting Time
 * Add TagCount and ExiryTime in the responses of getting propertites and downloading blobs
-* BlockBlob::Upload, AppendBlob::Create and PageBlob::Create are now by default non-overwriting
+
+### Breaking Changes
```

Comment:
Do the same for these:
https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/azure-storage-files-shares/BREAKINGCHANGES.md
https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/storage/azure-storage-files-datalake/BREAKINGCHANGES.md

Reply:
@katmsft